apps/editing-toolkit/package.json

```diff
@@ -74,8 +74,8 @@
 		"test:js:update-snapshots": "npx wp-scripts test-unit-js -u --config='bin/js-unit-config.js' --colors",
 		"test:e2e": "npx wp-scripts test-e2e --wordpress-base-url='http://localhost:4013' --colors",
 		"test:e2e:watch": "npm run test-e2e -- --watch",
-		"test:php": "npx wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/full-site-editing-plugin/phpunit.xml.dist'",
-		"clean": "npx rimraf dist full-site-editing-plugin/*/dist",
+		"test:php": "npx wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/editing-toolkit-plugin/phpunit.xml.dist'",
```

Comment:
`NB:` I can't seem to get the `test:php` command to run, but I'm marking this as non-blocking because I'm having the same trouble on the master branch

Reply:
@jeyip since checking out the branch have you:
- Run the build script?
- Started wp-env?

Reply:
Running the build script (`yarn build`) in `apps/editing-toolkit` fixed it. Thanks Noah!

Reply:
cool!